src/styles.scss

```
@@ -13,3 +13,23 @@ a {
   color: mat-color($primary);
   text-decoration: none;
 }
+
+.hljs {
+  padding: 0.5em;
+  border-radius: 8px;
+}
+
+// To avoid icon flickering
```

```suggestion
// to avoid icon flickering
```
But I'd rather prefer if we add some more context and info to this comment.